web_bench/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "scripts": {
+    "build:wasm-st": "wasm-pack build --target web",
+    "build:wasm-mt": "cross-env RUSTFLAGS=\"-C target-feature=+atomics,+bulk-memory,+mutable-globals\" rustup run nightly-2022-12-12 wasm-pack build --target web --out-dir pkg-parallel -- --features parallel -Z build-std=panic_abort,std",
```

Comment:
I'm missing rust-toolchain file that specifies this version of nightly?

Reply:
You propose to add rust-toolchain file and remove rustup invocation here?

Reply:
Added.